js/app.js

```diff
@@ -25,12 +25,214 @@ var firstandPike = {
       this.cookiesPerHour[i] = numberSold;
       this.dailySales += numberSold;
     }
+  },
+
+  renderSales: function (){
+    var ulElem = document.getElementById('salesinpike');
+    for (var i = 0; i < storeHours.length; i++){
+      //create an element
+      var listItemElem = document.createElement( 'li' );
+      //Give it a content
+      listItemElem.textContent = storeHours[i] + ': ' + Math.round(this.cookiesPerHour[i]) + ' cookies.';
+      //Append to the DOM
+      ulElem.appendChild(listItemElem);
+
+    }
+    listItemElem.textContent = 'Total of cookies: ' + Math.round(this.dailySales);
```

Comment:
Here, you don't create a new element, but instead modify your last `li`; this is why you don't have an 8pm `li` showing up, because you overwrite it with the title.